careers/faq.md

```diff
@@ -5,24 +5,40 @@ sidebar_label: FAQ
 
-#### Should I pay for co-op or just go for internships?
+Co-op students begin their journey in the Fall of 2nd year where they partake in a non-credit course called Transform to prepare you with job readiness skills such as resume writing, mock itnerviews, etc. In Winter of 2nd year, this is when students enter the Job Competition where they apply for jobs on mySuccess and for roles they find on their own. Summer of 2nd year is when students start their first work term. This default sequence can be changed to suit your needs, and we recommend reaching out to coop@uwindsor.ca for further information.
 
-Do both! Apply for internships and use co-op as a fallback. Once you've gotten your first co-op/internship it's
-significantly easier to get the next one on your own, and many people outgrow the co-op program.
+#### What is the difference between a Co-op and internship?
 
-#### Can I apply for internships while in the co-op program?
+Co-ops and internships are closely connected though have a few differences. The prime difference depends on if the employer requires you to be enrolled in a Co-op program. Co-op positions (and some internship opportunities) require you to be enrolled in a Co-op program, meanwhile most internship opportunities do not require you to be enrolled in a Co-op program. In essence, being enrolled in Co-op allows you to apply for both Co-op and internship opportunities, while not being enrolled in Co-op only limits you to applying to internship opportunities. Sometimes companies amy misuse the word internship when they actually mean Co-op, though to make sure, review the job description, and check if needing to be enrolled in a Co-op program is required.
 
-Co-op advisors advise against applying without their permission, but in the end there's nothing stopping you
-from applying for internships on your own. If you find one by yourself and want to stay in the co-op program,
-co-op will gladly accept that company as a co-op placement after a little screening.
+Up until now, the definition of Co-op and internship works best for Undergraduate and Master of Science students. However, Master of Applied Computing (MAC) students follow slightly different rules. Technically, MAC students are told they apply to internships only, but in reality, they can apply to both Co-op and internship roles as they are registered in a Co-op program. The reason that MAC students apply to "internships" relies on the second difference of Co-ops and internships; length of duration. Co-ops usually require 8 - 16 months of experience to graduate, though internships only require 4 months of experience. Undergradaute students require 3 work terms to graduate, and Master of Scieence students require 2 works term to graduate. However, MAC students only require 1 work term to graduate, hence why the Co-op office informs MAC students that they are applying for internships as the work term requirements are much shorter comapred to other programs.
```

Comment:
```suggestion
Up until now, the definition of Co-op and internship works best for Undergraduate and Master of Science students. However, Master of Applied Computing (MAC) students follow slightly different rules. Technically, MAC students are told they apply to internships only, but in reality, they can apply to both Co-op and internship roles as they are registered in a Co-op program. The reason that MAC students apply to "internships" relies on the second difference of Co-ops and internships; length of duration. Co-ops usually require 8 - 16 months of experience to graduate, though internships only require 4 months of experience. Undergraduate students registered in Co-op require 3 work terms to graduate, and Master of Science students require 2 works term to graduate. However, MAC students only require 1 work term to graduate, hence why the Co-op office informs MAC students that they are applying for internships as the work term requirements are much shorter compared to other programs.
```

```
 Co-ops usually require 8 - 16 months of experience to graduate, though internships only require 4 months of experience.
 ```
 does this mean co-op jobs or the co-op department? im not sure that it is accurate about co-ops requiring 8-16 months. I know it is more true for older institutions(like IBM, the government) but I would say its not completely true anymore.